java/src/jmri/jmrix/lenz/messageformatters/XNetMessageFormattersBundle.properties

```diff
@@ -0,0 +1,146 @@
+XNetReplyLIBaud                             = RESPONSE LI101 Baud Rate: {0}
+XNetReplyLIAddress                          = RESPONSE LI101 Address {0}
+XNetReplyCSStatus                           = Command Station Status:
+
+# XNetReply Command Station Status Bytes. These are concatenated together in the code.
+XNetCSStatusEmergencyOff                    = Emergency Off
+XNetCSStatusEmergencyStop                   = Emergency Stop
+XNetCSStatusServiceMode                     = Service Mode
+XNetCSStatusPoweringUp                      = Powering up
+XNetCSStatusPowerModeAuto                   = Auto power-up Mode
+XNetCSStatusPowerModeManual                 = Manual power-up Mode
+XNetCSStatusRamCheck                        = RAM check error!
+XNetCSStatusRamNormal                       = Normal Operations
+
+# XNetReply Loco information
+XNetReplyLocoStatus13Label                  = Locomotive F13-F28 Momentary Status:
+XNetReplyLocoNormalLabel                    = Locomotive Information Response: Normal Unit
+XNetReplyLocoMULabel                        = Locomotive Information Response: Locomotive in Multiple Unit
+XNetReplyLocoMUBaseLabel                    = Locomotive Information Response: Multi Unit Base Address
+XNetReplyLocoDHLabel                        = Locomotive Information Response: Locomotive in Double Header
+XNetReplyLoco2DHLabel                       = Second Locomotive in Double Header is:
+XNetReplyLocoLabel                          = Locomotive Information Response:
+XNetReplySearchNormalLabel                  = Search Response: Normal Locomotive:
+XNetReplySearchDHLabel                      = Search Response: Loco in Double Header:
+XNetReplySearchMUBaseLabel                  = Search Response: MU Base Address:
+XNetReplySearchMULabel                      = Search Response: Loco in MU:
+XNetReplySearchFailedLabel                  = Search Response: Search failed for:
+XNetReplyLocoOperated                       = is being operated by another device.
+XNetReplyFStatusLabel                       = Function Status:
+XNetReplyF13StatusLabel                     = F13-F28 Status:
+XNetReplyFeedbackLabel                      = Feedback Response:
+TurnoutWFeedback                            = Turnout with Feedback
+TurnoutWoFeedback                           = Turnout without Feedback
+XNetReplyNotOperated                        = Not Operated
+XNetReplyThrownLeft                         = Thrown Left
+XNetReplyThrownRight                        = Thrown Right
+XNetReplyInvalid                            = <invalid>
+XNetReplyMotionComplete                     = Motion Complete
+XNetReplyMotionIncomplete                   = Motion Incomplete
+XNetReplyFeedbackEncoder                    = Feedback Encoder Base Address:
+XNetReplyContactLabel                       = Contact:
+XNetReplyAddressInUse                       = Address in use by another device.
+XNetReplyAddressFree                        = Address is Free for Operation.
```

Comment:
This file doesn't have an empty line at the end of the file. We usually want that.

Reply:
fixed